src/main/scala/beam/agentsim/infrastructure/taz/TAZTreeMap.scala

```diff
@@ -163,8 +163,9 @@ class TAZTreeMap(val tazQuadTree: QuadTree[TAZ], val useCache: Boolean = false)
                   logger.warn(e.toString)
               }
               linkIdToTAZMapping += (id -> taz.tazId)
-            case _ =>
+            case None =>
               unmatchedLinkIds += id
+            case _ =>
```

Comment:
Matched, but not car/walk are just ignored then?

Reply:
Yep, exactly -- just ignoring them for the purposes of creating parking stalls